src/styles/main.scss

```diff
@@ -1,3 +1,4 @@
-body {
-  margin: 0;
-}
+@import "./utils/base";
+@import "./utils/variables";
+
+@import "./blocks/calendar.scss";
```

Comment:
Instead of importing calendar.scss directly into main.scss, create an import file (e.g., 'blocks/_imports.scss') for all block SCSS files and import that file into main.scss. This will help maintain a clean and organized structure.